lapis2-docs/src/components/MutationFilters/NucleotideMutations.astro

```diff
@@ -0,0 +1,16 @@
+---
+import { getReferenceGenomes, isMultiSegmented } from '../../reference_genomes';
+import SegmentNames from './SegmentNames.astro';
+
+const referenceGenomes = getReferenceGenomes();
+const multiSegmented = isMultiSegmented(referenceGenomes);
+---
+
+<p>
+    A nucleotide mutation has the format <code>{multiSegmented && '<segment>:'}{'<position><base>'}</code>.
```

Comment:
Could we not also have `<base><position><base>`, where the first base is optional, or do we leave it out, since we do not take it into account?
(Same for amino acids)

Reply:
We simply ignore the "from base", so I think I would simply leave it out here...